src/v3/src/types/schema.ts

```diff
@@ -216,16 +215,16 @@ export interface DescriptionElement extends UISchemaElement {
 
 export interface TextWithHtmlElement extends UISchemaElement {
   type: 'TextWithHtml';
-  options: DescriptionElement['options'] & {
+  options: ActionOptions & {
+    content: string;
     className: string;
-    step: string;
     stepToRender?: string;
   };
 }
 
 export interface ReminderElement extends UISchemaElement {
   type: 'Reminder';
-  options: {
+  options: ActionOptions & {
```

Comment:
Change `ctaText` to `content` (and make new prop specific below, something like `isContentHtml` or `contentHasHtml`

Reply:
update `linkLabel` to `buttonText`